modules/vactory_academy_agency/src/Controller/FormationInscriptionController.php

```diff
@@ -101,7 +101,7 @@ public function subscribeToFormationConfirmation(Request $request) {
                 $datalayer_attributes[$key] = 'Ancien client';
               }
               elseif (in_array($key, $client_infos_fields)) {
-                $datalayer_attributes[$key] = "${$key}";
+                $datalayer_attributes[$key] = "{$$key}";
```

Comment:
@mhdelazhari this works ?

Reply:
@voidagency that's what I found while searching
![image](https://user-images.githubusercontent.com/102218719/211084692-f1fae921-3e06-4a6d-afe0-ad5543e4b233.png)